.github/workflows/tpu_ci.yml

```diff
@@ -0,0 +1,33 @@
+name: TPU Integration Test
+run-name: TPU Testing
+on:
+  workflow_dispatch:
+  schedule:
```

Comment:
Can you run this on push to `master` instead of on a schedule?

Reply:
Done